packages/angular/src/directives/navigation/ion-router-outlet.ts

```diff
@@ -39,22 +43,30 @@ import { RouteView, getUrl } from './stack-utils';
 // eslint-disable-next-line @angular-eslint/directive-class-suffix
 export class IonRouterOutlet implements OnDestroy, OnInit {
   nativeEl: HTMLIonRouterOutletElement;
-
-  private activated: ComponentRef<any> | null = null;
   activatedView: RouteView | null = null;
+  tabsPrefix: string | undefined;
 
-  private _activatedRoute: ActivatedRoute | null = null;
   private _swipeGesture?: boolean;
-  private name: string;
   private stackCtrl: StackController;
 
   // Maintain map of activated route proxies for each component instance
   private proxyMap = new WeakMap<any, ActivatedRoute>();
-
   // Keep the latest activated route in a subject for the proxy routes to switch map to
   private currentActivatedRoute$ = new BehaviorSubject<{ component: any; activatedRoute: ActivatedRoute } | null>(null);
 
-  tabsPrefix: string | undefined;
+  private activated: ComponentRef<any> | null = null;
+  /** @internal */
+  get activatedComponentRef(): ComponentRef<any> | null {
+    return this.activated;
+  }
+  private _activatedRoute: ActivatedRoute | null = null;
+
+  /**
+   * The name of the outlet
+   *
+   * @see [named outlets](guide/router-tutorial-toh#displaying-multiple-routes-in-named-outlets)
```

Comment:
Where is this supposed to link to?

Reply:
This is copy/paste from Angular's source. I'll remove that extra link, but keep the comment block.